### DISCUSSION TITLE
Core skills reset on Forge update

### DISCUSSION THREAD

**Discussion by danrcn** [Mezryss/FVTT-Genesys#102] [Q&A] [answered]
Hi! I've been setting up to run Gene-SWRPG in this module (which is excellent! Thank you so much for all your work!) using The Forge to host, and it seems like my Core Skills have been reset, reverting a couple hours of prep work :(

I had customized my list of skills to include some changes (custom Force and Knowledge skills, dropping unnecessary ones like Riding and Melee Light/heavy, etc). I made all those changes to a skill compendium, then used the Module Settings tab to select that compendium as the active one. Boom, sorted. Worked great, I started making character sheets.

I logged in a week later and found my core skills compendium had reset to the Core Rules version. I imagine this is because the module updated on the back end (I do have "update to latest stable version" checked in my Forge game manager), but the part that confuses me is that the Module setting to select Core Skills compendiums is gone, and now only accepts a filename input. I remember previously there was a button that gave me the ability to download, edit, and re-upload skills lists. To add to my confusion, the filename listed (for the default skills) is "genesys.crb-skills" not "Core Skills" as it's listed on the Compendium browser tab. 

My version number is 0.1.15, if that matters. I don't know what it was before.

So, my direct questions:
Is there a way to find my previous core skills compendium, or is it gone forever because of the auto-update? I still have the text in-game (character sheets created previously are fine) so manually copy-pasting it will probably work, but if I can avoid this I would prefer it.
To avoid this happening again, how do I use a custom skills compendium now? 

Thanks in advance for your help!

**Comment by Assembling-Kings** [comment, marked as answer]
I have to admit this is pretty strange. The system setting for specifying the skills compendium has been a filename for as long as I can remember (I've contributed to this project since March of this year). So from my perspective it's more likely that a module is responsible for whatever is happening.

> Is there a way to find my previous core skills compendium, or is it gone forever because of the auto-update?

If you create it as a module then it should appears in your modules tab. If you backed up those you might find a copy there. If you made it as a compendium directly in the world then it should be there. If not then it seems like the world regressed somehow.

> To avoid this happening again, how do I use a custom skills compendium now?

Same as before. Here's some guidance with regards to that (check 'Starting with Skills'): https://github.com/Mezryss/FVTT-Genesys/discussions/50
